cluster/gce/gci/configure.sh

```diff
@@ -644,6 +683,16 @@ function install-kube-binary-config {
     log-wrap "RemountFlexVolume" remount-flexvolume-directory "${VOLUME_PLUGIN_DIR}"
   fi
 
+  # When ENABLE_AUTH_PROVIDER_GCP is set, following flags for out-of-tree credential provider for GCP
+  # are presented to kubelet:
+  # --feature-gates=DisableKubeletCloudCredentialProviders=true,KubeletCredentialProviders=true
```

Comment:
This comment regarding feature gate being enabled automatically with `ENABLE_AUTH_PROVIDER_GCP` is no longer true (maybe note that it is required that those two feature gates should be enabled)